src/libs/tools/global_toolbox.c

```diff
@@ -633,7 +638,10 @@ static void _main_do_event(GdkEvent *event, gpointer data)
               }
             }
             if(!is_language_supported) lang = "en";
-            char *url = g_build_path("/", base_url, lang, help_url, NULL);
+            char *url = dt_is_dev_version()
+              ? g_build_path("/", base_url, help_url, NULL)
+              : g_build_path("/", base_url, lang, help_url, NULL);
```

Comment:
At some point, for the non-dev version we'd like to put the version number (X.Y) in the URL. If we can get darktable.org transferred to a new server soon enough I hope we can set this up early for testing.

Reply:
Ok, I'll wait for this before merging then.